sample/app/build.gradle

```diff
@@ -0,0 +1,156 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:2.2.0'
+        // To depend on the deployed version of the plugin add the following line here:
+        // classpath 'com.novoda:gradle-android-command-plugin:1.3.0'
```

Comment:
I thought that the composite build feature of Gradle would allow us specify the configuration as intented, and the only override it for development. 
Is that due to the Gradle version that we do not use that here?